### ISSUE TITLE
Add a task name to the web notifications list (instead of just a #id - as it is now)

### ISSUE THREAD

**Issue by temp5mr** [kanboard/kanboard#5456]
I just slightly adjusted the code in **templates\web_notifications\show.php** and we all really liked it. Perhaps it's not the most correct approach, but it would be great to have such a feature properly implemented.
This part:
```
        <span class="table-list-title">
            <?php if ($this->text->contains($notification['event_name'], 'subtask')): ?>
             ...
            <?php endif ?>

			<?php if (isset($notification['event_data']['task']['project_name'])): ?>
				<?= $this->url->link(
					$this->text->e($notification['event_data']['task']['project_name']),
					'BoardViewController',
					'show',
					array('project_id' => $notification['event_data']['task']['project_id'])
				) ?> &gt;
			<?php elseif (isset($notification['event_data']['project_name'])): ?>
				<?= $this->text->e($notification['event_data']['project_name']) ?> &gt;
			<?php endif ?>

			<?php
			
			if (isset($notification['event_data']['task'])):
				$taskID = $notification['event_data']['task']['id'];
				$taskTitle = $notification['event_data']['task']['title'];

				if ($this->text->contains($notification['event_name'], 'task.overdue') && count($notification['event_data']['tasks']) > 1):
					echo "Задача #$taskID: $taskTitle"; 
				else:
					echo $this->url->link(
						"Задача #$taskID: $taskTitle",
						'WebNotificationController',
						'redirect',
						array('notification_id' => $notification['id'], 'user_id' => $user['id'])
					); 
				endif;
			endif;
			?>
        </span>
```

**Comment by tomeli5n** [comment]
Hello, 

This week I made a plugin to show this detail to the user. 

https://github.com/tomeli5n/NotifyPlus

**Comment by h-e-l-l-o-w-o-r-l-d** [comment]
In my opinion the user notifications are the weakest point in Kanboard, Most of our users don't use it because the information is too basic and not grouped by task. On the other hand, emails are not sent out from a queue, but on the fly, which nobody likes. So we were a bit lost here.

Because of this and because no plugin existed, I also started to code a notification plugin 1 week ago, so I was a bit shocked to see this now. :o) I don't like the idea of having 2 plugins for the same purpose, but looking at your Plugin, my version is more advanced with all information you can think of like column changes, highlighted mentions and so on. Also, it is closer to the core.

So I will continue and finish it, which will take approx 2-3 weeks. My plan was to release it on Github and ask @fguillot if he would prefer a pull-request.

I also planned the mentioned (optional) email queue functionality, but I'm not sure yet how to properly implement it.

Stay tuned

@tomeli5n Looking at the screenshots... which CSS-plugin do you use to have such nice and clean styles?

**Comment by tomeli5n** [comment]
> @tomeli5n Looking at the screenshots... which CSS-plugin do you use to have such nice and clean styles?

[ThemeRevision Theme](https://github.com/greyaz/ThemeRevisionhttps://github.com/greyaz/ThemeRevision)

**Comment by tomeli5n** [comment]
> Feel free to submit a pull-request. It's probably better to have this enhancement directly in Kanboard instead of having an external plugin.

I’m still resolving some uses cases with the plugin since it currently only groups notifications by task.

As soon as I can handle all issues and notification types, I will generate a pull request for review. For example, a notification about multiple overdue tasks on a board is not associated with any particular task.
